src/main/java/lembas/core/LembasCore.java

```diff
@@ -0,0 +1,130 @@
+/*******************************************************************************
+ * Copyright (C) 2019 Lembas Modding Team
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ ******************************************************************************/
+package lembas.core;
+
+import cpw.mods.fml.common.*;
+import cpw.mods.fml.common.Mod.*;
+import cpw.mods.fml.common.event.*;
+import lembas.core.api.client.LembasCoreClient;
+import lembas.core.api.server.LembasCoreServer;
+import lembas.core.internal.client.LembasCoreClientImpl;
+import lembas.core.internal.common.LembasCoreCommonImpl;
+import lembas.core.internal.server.LembasCoreServerImpl;
+
+/**
+ * The main mod file of Lembas Core. Everything which can be accessed from the
+ * outside (modid, version, getClient(), ...) is considered to be a part of the
+ * API. Internal implementation details (for example the proxy implementation
+ * classes) ARE NOT PART OF THE API.
+ * 
+ * @author CraftedMods

```

Comment:
Honestly, this is generated by my IDE, and I didn't change it. I would say be use that so we see who created the file (which normally should be the person who has the most knowledge about it), but as the file evolves, this can be changed/removed. On the other hand, git just lets us see that too. We can remove it, if it bothers you.

Reply:
Let us just await @VinyarionHyarmendacil’s (and @AlteOgre’s) reply.

Reply:
Adressed with 96c98a7c168d8cdb40a3503d99163b9cb97a5f81